### PR TITLE
Created individual API files for requests

### PR DESCRIPTION
**Pull request by CatalinTiu**

### CODE REVIEW

File: src/api/meetings.ts [hunk 1]
```diff
@@ -17,4 +18,13 @@ const addMeeting = async (
     });
 };
 
-export { addMeeting };
+const getMeetingsData = async () => {
```
Comment (robert-szalkai) [inline src/api/meetings.ts:21]: add return type for this fn

File: src/api/participants.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+import { Participant } from "../interfaces/interfaces";
+
+const getParticipantsIdName = async () => {
```
Comment (robert-szalkai) [inline src/api/participants.ts:5]: same here, return type

**Review by robert-szalkai — APPROVED** [review]

File: src/api/rooms.ts [hunk 3]
```diff
@@ -1,4 +1,17 @@
 import axios from "axios";
+import { iRoomCards } from "../interfaces/interfaces";
+
+const getRooms = async () => {
```
[the remark this answers is not in the document]
Comment (CatalinTiu) [reply src/api/rooms.ts]: Adde return type

File: src/api/participants.ts (hunk 2 above)
Comment (CatalinTiu) [reply src/api/participants.ts]: Added return type

**Pull request merged**